### DISCUSSION TITLE
我进来了哦

### DISCUSSION THREAD

**Discussion by dc29120** [share121/inter-knot#1888] [其他]
![357088285-ec6edc0b-f68f-48f3-8b44-5ea75efe863f](https://github.com/user-attachments/assets/6769414d-dc4b-4295-af72-b1f2f6653180)
挺简单的，只要是Edge，浏览器下的扩展，创建了新的github账号，就轻轻松松进来了。

**Comment by jiangshen-N** [comment]
哦咩得多

**Comment by AKT505** [comment]
哦咩得多

**Comment by Drjiedian** [comment]
我是Edge，下了个叫篡改猴的扩展，然后就能下绳网脚本了😗（有人告诉我GIT HUB怎么发表情嘛，Windows自带的真特么难看）

**Comment by gaojiaobei** [comment]
哦咩得多

**Comment by 5888269** [comment]
哪个扩展

**Reply by Drjiedian** [reply]
篡改猴，这是我下的，上面有回复

**Reply by dc29120** [reply]
你打开绳网时，它会让你跳转过去的

**Comment by 5888269** [comment]
打不开
![Uploading Screenshot_20240813_191549.jpg…]()

**Reply by luo-luo-o** [reply]
看到`uploading`说明图片还在上传，要等上传完了才发送哦！！不然是看不到图片的

**Comment by liyinlin0128** [comment]
帅

**Comment by Jshuya24** [comment]
贤者之桶怎么跑出来了？
![Uploading 02F293A276E52C3D907AAC6D24D0EAE3.jpg…]()

**Comment by 566001** [comment]
国米娜赛

**Comment by AnbulesPeo** [comment]
哦咩得多

**Comment by lllliufeng** [comment]
哦次咔嘞撒嘛嘚洗哒

**Comment by 1451374815** [comment]
edge总是在奇奇怪怪的地方好用
